Dockerfile.wmg_pipeline

```diff
@@ -4,7 +4,7 @@ FROM python:3.10
 RUN /usr/local/bin/python -m pip install --upgrade pip && \
     apt update && apt -y install graphviz graphviz-dev && \
     rm -rf /var/lib/apt/lists/* && \
-    pip3 install awscli
+    pip3 install awscli==1.32.24
```

Comment:
if this will be pinned then we should pin it in the requirements.txt file, that way dependabot can check it for us.

Reply:
This is a CLI command and not a library, so I think this is the right place. In fact, I would recommend to install it with the method suggested [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).